Source/PropertyPolicy.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  PropertyPolicy.swift
+//  KakapoExample
+//
+//  Created by Alex Manzella on 31/03/16.
+//  Copyright © 2016 devlucky. All rights reserved.
+//
+
+import Foundation
+
+public protocol _PropertyPolicy: Serializable {
+    var _object: Any { get }
+    var shouldSerialize: Bool { get }
+}
+
+public protocol PropertyPolicy: _PropertyPolicy {
+    typealias T
+    var object: T? { get }
+    var shouldSerialize: Bool { get }
+}
+
+extension PropertyPolicy {
+    public var _object: Any {
+        get {
+            return object
+        }
+    }
+}
```

Comment:
You would just return 0 if nil and the object if not nil, type must be Any if the object is not Int